driver/post_remove.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# Bash script to remove the old module f18601 and install the new one generated by dkms
```

Comment:
Does this comment need to be updated? The f18601 is different hardware, right?

Reply:
Absolutly, it's from fintek that was the first dkms we didi and that I still use as a template. Thx!

Reply:
0ab3d51f5d43056df3d6281a6a1ced672bac76c8